src/modules/cluster/cluster-list/cluster-list.html

```diff
@@ -64,8 +64,7 @@ <h1 class="bold-text">Clusters</h1>
             </div>
             <div class="col-md-1 text-center cluster-name bold-text counts" ng-click="clusterCntrl.goToClusterDetail(cluster.id)">{{cluster.name}}</div>
             <div class="col-md-3">
```

Comment:
@kamleshverma1 as we discuss in the morning call. Please pass the id's or classes to the component for behavior test.